### DISCUSSION TITLE
Incompatible Layers

### DISCUSSION THREAD

**Discussion by giacomobandinelli** [nftchef/art-engine#75] [Help + Q&A]
I want to use the incompatible layers function, but I want to make 2 traits from 2 different layers incompatible.

For example, in the Background layer there is a Gold Background trait and I want it to be incompatible with a Silver Hat trait from the Hats layer.
How do I change this? 

`const incompatible = {
  Red: ["Dark Long"],
  "Name with spaces": ["buzz","rare-Pink-Pompadour" ]
  // directory incompatible with directory example
  White: ["rare-Pink-Pompadour"],
};`

**Comment by nftchef** [comment]
hi! 
If I understand correctly, this is what `incompatible` is used for. It _only_ works when mapping a trait from one folder to one or more traits from _a different_ folder.

in you example, you would have 
```js
incompatible = {
    "Gold Background": ["Silver Hat"]
}
```
And for more than one, you can add to the array like, 
```js
incompatible = {
    "Gold Background": ["Silver Hat", "Bronze Hat", "Pink Hat"]
}
```

For more complex combinations and if/then situations, nested folder branching should be used.
